mrc/localization/color/utils/circle_points.py

```diff
@@ -2,7 +2,20 @@
 
 
 class CirclePoints:
+    """
+    Class responsible for helping to iterate through circle points.
```

Comment:
Class responsible for getting all points inside a circle? Or sth similar, I think.

Reply:
Done